kit/preprocess.js

```diff
@@ -133,13 +133,14 @@ export const frameworkcontentPreprocess = {
 		const REGEX_PYTORCH = /<pt>(((?!<pt>).)*)<\/pt>/ms;
 		const REGEX_TENSORFLOW = /<tf>(((?!<tf>).)*)<\/tf>/ms;
 		const REGEX_JAX = /<jax>(((?!<jax>).)*)<\/jax>/ms;
-		const FRAMEWORKS = [
-			{ framework: "pytorch", REGEX_FW: REGEX_PYTORCH, isExist: false },
-			{ framework: "tensorflow", REGEX_FW: REGEX_TENSORFLOW, isExist: false },
-			{ framework: "jax", REGEX_FW: REGEX_JAX, isExist: false }
-		];
-
+		
 		content = await replaceAsync(content, REGEX_FRAMEWORKCONTENT, async (_, fwcontentBody) => {
+			const FRAMEWORKS = [
```

Comment:
the bug was being caused where `FRAMEWORKS` const was a "global" state rather than "local" (i.e. inside the closure)